janus-idp/operator
# Fix formatting issues with default config ConfigMap

## Description
Kustomize was rendering render the content of the default config ConfigMap as raw string, making it hard to read and edit.
This PR fixes that by using different delimiters for the placeholder strings.

## Which issue(s) does this PR fix or relate to

- Fixes #89 

## PR acceptance criteria


## How to test changes / Special notes to the reviewer
See #89 for more context.
`make bundle` should now produce proper formatting for `db-statefulset.yaml` and `deployment.yaml` ConfigMap keys.
